### PR TITLE
ANC PostPartum - Specialty Care

### PR DESCRIPTION
**Pull request by anvmn**
#376
#377

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-08-18 11-44

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/SyncManager/View.elm [hunk 1]
```diff
@@ -618,6 +618,9 @@ viewAuthorityEntity backendAuthorityEntity =
             BackendAuthorityPrenatalSendToHC identifier ->
                 viewMeasurement identifier "Prenatal Send to HC"
 
+            BackendAuthorityPrenatalSpecialityCare identifier ->
+                viewMeasurement identifier "Prenatal Speciality Care"
```
Comment (mariano-dagostino) [inline client/src/elm/SyncManager/View.elm]: `Specialty`

File: server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.features.field_instance.inc [hunk 2]
```diff
@@ -6747,6 +6747,318 @@ function hedley_prenatal_field_default_field_instances() {
...
+      'teaser' => array(
+        'label' => 'above',
+        'settings' => array(),
+        'type' => 'hidden',
+        'weight' => 0,
+      ),
+    ),
+    'entity_type' => 'node',
+    'field_name' => 'field_speciality_care_signs',
+    'label' => 'Speciality Care Signs',
```
Comment (mariano-dagostino) [inline server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.features.field_instance.inc]: `Specialty`

File: server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.features.field_instance.inc [hunk 3]
```diff
@@ -8926,6 +9238,7 @@ function hedley_prenatal_field_default_field_instances() {
   t('Send to HC');
   t('Shards');
   t('Specialist at HC');
+  t('Speciality Care Signs');
```
Comment (mariano-dagostino) [inline server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.features.field_instance.inc]: `Specialty`

File: server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.features.inc [hunk 4]
```diff
@@ -147,6 +147,14 @@ function hedley_prenatal_node_info() {
       'title_label' => t('Title'),
       'help' => '',
     ),
+    'prenatal_speciality_care' => array(
+      'name' => t('Prenatal Speciality Care'),
```
Comment (mariano-dagostino) [inline server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.features.inc]: `Specialty`

File: server/hedley/modules/custom/hedley_restful/plugins/restful/node/activity/prenatal-speciality-care.inc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Restful plugin.
+ */
+
+$plugin = array(
+  'label' => t('Prenatal Speciality Care'),
```
Comment (mariano-dagostino) [inline server/hedley/modules/custom/hedley_restful/plugins/restful/node/activity/prenatal-speciality-care.inc]: `Specialty`

File: server/hedley/modules/custom/hedley_restful/plugins/restful/node/activity/prenatal-speciality-care.inc [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  'resource' => 'prenatal-speciality-care',
+  'name' => 'prenatal-speciality-care',
+  'entity_type' => 'node',
+  'bundle' => 'prenatal_speciality_care',
+  'description' => t('Exports the Prenatal Speciality Care bundle.'),
```
Comment (mariano-dagostino) [inline server/hedley/modules/custom/hedley_restful/plugins/restful/node/activity/prenatal-speciality-care.inc]: `Specialty`

**Review by mariano-dagostino — CHANGES_REQUESTED** [review]
@anvmn Looks good. I found several places where user facing `Speciality` strings needs to be changed to `Specialty`

**Comment by anvmn** [comment]
All review comments addressed. Merging...

**Pull request merged**